packages/libs/react-ui/src/components/Form/Select/Select.tsx

```diff
@@ -34,13 +35,16 @@ export const Select: FC<ISelectProps> = forwardRef<
   { ariaLabel, children, disabled = false, outlined = false, icon, ...rest },
   ref,
 ) {
+  const { status } = useContext(InputWrapperContext);
+
   const Icon = icon && SystemIcon[icon];
   const ChevronDown = SystemIcon.ChevronDown;
 
   return (
     <div
       className={classNames(containerClass, {
         [containerClassDisabled]: disabled,
+        [statusClass]: status,
         [baseOutlinedClass]: outlined,
```

Comment:
Previously in the Input component I saw you were able to apply the status colors using the baseOutlinedClass. Can you do that here too?

Reply:
yes

Reply:
yes